src/js/components/Comments/CommentAdd.vue

```diff
@@ -25,8 +25,10 @@
 		<user-div :user-id="currentUser" />
 
 		<form class="commentAdd" name="send-comment">
-			<input v-model="comment" class="message" data-placeholder="New Comment ...">
-			<input v-show="!isLoading" class="submit-comment icon-confirm" @click="writeComment">
+			<input v-model="comment" class="message" data-placeholder="New Comment ..."
+				@keyup.enter="writeComment">
+			<input v-show="!isLoading" class="submit-comment icon-confirm"
+				@click="writeComment">
```

Comment:
Is there a reason you are using an input field as button?

If not, you could do something like this:
```vue
<form class="..." name="..." @submit="writeComment">
    <input v-model="comment" class="message" data-placeholder="New Comment ..." />
    <button type="submit" v-show="!isLoading" class="icon-confirm">...</button>
</form>
```